portafly/src/i18n/locales/en/audience/account/accounts.yml

```diff
@@ -20,13 +20,13 @@ applications_description: This is the title of the tab containing the list of ap
 # Tab overview
 
 organization_label: Organization/Group Name
-organization_value: {{organization_name}}
+organization_value: "{{organization_name}}"
```

Comment:
@damianpm Do we also need quotation marks for standalone variables?

Reply:
As previously said, these are unusable for us, so it doesn't really matter.

Reply:
Removed!